pyproject.toml

```diff
@@ -7,7 +7,11 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-Flask = "^1.0"
+Flask = [
```

Comment:
express Flask dependency conditionally (depending on the installed version of Python)

Reply:
I think this solves point 6 that you mentioned here: https://github.com/jwg4/flask-selfdoc/pull/72#issuecomment-1435263813

However it is still not possible to rely on the own dependencies config of Flask 1.X because it accepts too recent versions of `itsdangerous`, `Jinja2` and `MarkupSafe` that cause `ImportError` etc.
So these have to be manually overwritten with older, compatible versions like I did in `test.yml`